app/uk/gov/hmrc/tai/config/ApplicationConfig.scala

```diff
@@ -74,7 +74,10 @@ class ApplicationConfig @Inject()(
   lazy val cocarFrontendUrl = s"$cocarFrontendHost/paye/company-car/details"
   lazy val marriageServiceHistoryUrl = s"$tamcFrontendHost/marriage-allowance-application/history"
   lazy val medBenefitServiceUrl = s"$benefitsFrontendHost/paye/benefits/medical-benefit"
-  lazy val sa16UpliftUrl = s"$identityVerificationHost/mdtp/uplift"
+
+  lazy val ivUpliftprefix = decorateUrlForLocalDev("identity-verification.prefix")
```

Comment:
@vimalsuresh uplift should now direct you to iv stubs for local and staging, everything else is real mdtp, this might break an AT

Reply:
👍